CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### fixed
+
+- rounded button style (https://github.com/openfun/marsha/pull/2118)
```

Comment:
```suggestion
- rounded button style (#2118)
```

This is enough IMO.

Reply:
Doesn't seem: https://github.com/openfun/marsha/blob/73a80000a391f12b7eca903026ffb1158f2aa17c/CHANGELOG.md

Reply:
Yes but I think it's enough. Adding a full link disable the linting on this line.